src/Glpi/Http/Firewall.php

```diff
@@ -201,4 +205,44 @@ private function normalizePath(string $path): string
         $path = preg_replace('/\\\/', '/', $path);
         return $path;
     }
+
+    private function computeForPaths(string $path): ?string
+    {
+        if (isset($_GET["embed"], $_GET["dashboard"]) && str_starts_with($path, '/front/central.php')) {
+            return 'no_check';
```

Comment:
We should keep existing comments. They may help developpers to understand why there is a special case here.
```suggestion
            // Allow anonymous access for embed dashboards.
            return 'no_check';
```